docs/notebooks/environment.yml

```diff
@@ -5,13 +5,14 @@ channels:
 dependencies:
  - python==3.8
  - numpy>=1.14.0
- - pandas>=1.1.2
- - anndata==0.7.8
+ - pandas>=1.5.3
+ - anndata==0.8.0
```

Comment:
these versions were incompatible and the new scripts were erroring out on the step where I call `optimize_adata`